### ISSUE TITLE
Reason behind the new ZeroPadding2D layer in updated AUTOMAP

### ISSUE THREAD

**Issue by jainspoornima** [MattRosenLab/AUTOMAP#7]
I wish to know the motivation behind adding the zero-padding layer to increase size to 72x72 from 64x64 in updated AUTOMAP. It increases the size before the first convolutional layer C1. Since it has padding="same", it will anyways pad the input based on the kernel size before convolving.

**Comment by danyalb** [comment]
It was done to avoid any edge effects the convolutional layers and tensorflow might add. As shown in the loss function in the code and as was done for the original paper, these zero-padded layers were removed before the loss function.
